src/components/tree/phyloTree/change.ts

```diff
@@ -242,6 +248,11 @@ export const modifySVGInStages = function modifySVGInStages(
     this.updateTipLabels();
     this.drawTips();
     if (this.vaccines) this.drawVaccines();
+    if (this.measurementsColorGrouping) {
+      this.drawMeasurementsColoringCrosshair();
+    } else {
+      this.removeMeasurementsColoringCrosshair();
+    }
```

Comment:
For `modifySVGInStages` you'll want to remove the crosshair in step 1 and redraw it in step 3. Currently it stays in place while the tree tips move and then jumps to its new location

(Overall the PR looks really good)